src/main/java/at/hannibal2/skyhanni/utils/LorenzRarity.kt

```diff
@@ -45,12 +46,19 @@ enum class LorenzRarity(val color: LorenzColor, val id: Int) {
         return rarityBelow
     }
 
+    fun getCleanName(): String {
```

Comment:
this is just the same as our `.firstLetterUppercase()` function in stringutils, although i think this one may be better. either use that function or change the one in stringutils